check/fixes.frm

```diff
@@ -1694,7 +1694,25 @@ P;
 .end
 assert succeeded?
 assert result("F") =~ expr("rat(1,ep^2 + 2*ep + 1)")
-*--#] Issue180 : 
+*--#] Issue180 :
+*--#[ Issue183 :
+#: MaxTermSize 16K
+#: SubTermsInSmall 800
+
+CF f;
+Auto S x;
+L F = f(<x1+x2+x3+x4>,...,<x6+x7+x8+x9>);
+*repeat id f(x1?,x2?,?a) = f(x1*x2,?a);  * error: Sorted function argument...
+transform f,mulargs(1,last);  * silent crash
+P;
+.end
+# Runtime errors may freeze ParFORM.
+#pend_if mpi?
+assert runtime_error?
```

Comment:
Here you can write: `assert runtime_error?("Term too complex during normalization")`.